### PR TITLE
GUI

### PR DESCRIPTION
**Pull request by Joshua-Crocker**
Finished the inventory and checkout window.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from PyQt6 import uic
 from PyQt6.QtWidgets import *
+from PyQt6.QtCore import *
```
Comment (Parker-Wallace) [inline main.py:5]: does this need to be added to the requirements documentation?

File: main.py [hunk 2]
```diff
@@ -107,19 +178,20 @@ def pdelBtnClickHandler(self):
...
             if button == QMessageBox.StandardButton.Yes:
                 pid = self.pidLineEdit.text()
-                result = deleteProductById(pid)
+                result = deleteProduct(pid)
                 if result == 1:
+                    self.lblEditDeleteProd.setText('Product deleted successfully!')
                     self.refreshProductTable()
                     self.refreshProductComboBox()
                 else:
-                    print('Skill issue')
```
Comment (Parker-Wallace) [inline main.py:115]: thanks for this correction

**Review by Parker-Wallace — APPROVED** [review]
can you use chat gpt or somthing to add documentation to the code? will merge onto main if not and add throughout the week

File: main.py (hunk 1 above)
Comment (Dylan-M-afk) [reply main.py:5]: Should be fine I believe

**Comment by Dylan-M-afk** [comment]
The changes look good to me.

**Pull request merged**